### PR TITLE
Improve vendor handling:

### LINKED ISSUE #73: Vendor Handling Improvements
Improvements for the vendor handling are necessary:

**1.** if there is a creation of a release and a vendor is to be set, the user usually searches for a vendor. However, if the desired is not among the list, but needs to be created, there is not means for doing so. Instead the user needs to make a more special search to get to the button that allows him to create one.

**2.** Vendor short name, vendor ID  and vendor name seem to be mixed in the Vendor portlet and the search mask when adding a vendor to a release. (See screenshot)

036 oss sw360 20160120 screenshot vendor mismatch 01
036 oss sw360 20160120 screenshot vendor mismatch 02

**3.** It should be possible to delete a vendor in the vendor detail view.

### LINKED ISSUE #124: Vendor portlet is broken
#### Problem:
on current master (5d521b5ab31bcf106cf5e1d65daa354f3da3fe2e, clean vagrant build) is the vendor portlet broken.

#### What was done:
**1.** installation and setup on new box
**2.** log in as `admin`
**3.** go to vendor portlet ([.../group/guest/vendors](https://localhost:8443/group/guest/vendors))

**Situation:** At this point everything looks fine, i.e. an empty table is shown.

**4.** create an arbitrary vendor
**5.** go back to vendor portlet ([.../group/guest/vendors](https://localhost:8443/group/guest/vendors))

**Situation:** Now there is now table rendered at all (see screenshot).

#### other observations
In the search portlet one is able to find the created vendor and also the data in the database looks fine.

The problem might be, that there is a problem with `PortletURL`, i.e.
```
vendors:51 Uncaught TypeError: Cannot read property 'createURL' of undefined
```

in the code 
```javascript
50:    function createUrl_comp( paramId, paramVal) {
51:        var portletURL = Liferay.PortletURL.createURL( baseUrl ).setParameter(pageName,pageEdit).setParameter(paramId,paramVal);
52:        return portletURL.toString();
53:    }
```

--
The vendor portlet without list:
tmp
--
The error mentioned above:
tmp

### PR DESCRIPTION
**Pull request by heydenreich**
- add button always present in releases-set vendor
 - vendor list in releases-set vendor with consistent names
 - Vendors throughout SW360 always in order Full Name, Short Name, URL
 - Delete Function in vendor/edit.jsp
 - fixed broken vendor table in vendor/view.jsp (links to edit-pages were broken)
 - closes #73
 - closes #124

### CODE REVIEW

File: frontend/sw360-portlet/src/main/webapp/html/components/includes/vendors/addVendorForm.jspf [hunk 1]
```diff
@@ -19,28 +19,27 @@
...
-
-      <tr>
-        <td >
-          <label class="textlabel stackedLabel mandatory" for="addVendorUrl">URL</label>
-          <input id="addVendorUrl" type="url" placeholder="Enter Vendor url" required="">
-        </td>
-      </tr>
-
 
+        <tr>
```
Comment (maximilianhuber) [inline frontend/sw360-portlet/src/main/webapp/html/components/includes/vendors/addVendorForm.jspf]: Just indentation changes?

File: frontend/sw360-portlet/src/main/webapp/html/admin/vendors/edit.jsp [hunk 2]
```diff
@@ -49,7 +53,11 @@
 
 <div id="where" class="content1">
     <p class="pageHeader"><span class="pageHeaderBigSpan"><sw360:out value="${vendor.fullname}"/></span>
-
+        <core_rt:if test="${not addMode}" >
+            <input type="button" class="addButton" onclick="window.location.href='<%=deleteVendorURL%>'"
+                   value="Delete <sw360:out value="${vendor.fullname}"/>"
+            >
```
Comment (maximilianhuber) [inline frontend/sw360-portlet/src/main/webapp/html/admin/vendors/edit.jsp]: It is very uncommon to put this into it's own line

**Comment by mcjaeger** [comment]
tested and could be merged (aware of the formatting issues, if you pan to change this, so I am not merging)

**Comment by maximilianhuber** [comment]
There are also other small issues. I will merge it, if they are solved.


--Maximilian Huber * maximilian.huber@tngtech.com * +49-174-3410223
TNG Technology Consulting GmbH, Betastr. 13a, 85774 Unterföhring
Geschäftsführer: Henrik Klagges, Christoph Stock, Dr. Robert Dahlke
Sitz: Unterföhring * Amtsgericht München * HRB 135082-------- Ursprüngliche Nachricht --------Von: Michael <notifications@github.com> Datum: 17.03.2016  18:06  (GMT+01:00) An: siemens/sw360portal <sw360portal@noreply.github.com> Cc: Maximilian Huber <maximilian.huber@tngtech.com> Betreff: Re: [sw360portal] Improve vendor handling: (#126) 
tested and could be merged (aware of the formatting issues, if you pan to change this, so I am not merging)

**Pull request merged**